eslint.config.mjs

```diff
@@ -127,7 +127,12 @@ export default [
       parser: typescriptEslint.parser,
       parserOptions: {
         allowAutomaticSingleRunInference: true,
-        projectService: true,
+        projectService: {
+          allowDefaultProject: [
+            "packages/babel-helpers/src/helpers/applyDecs2305.ts",
```

Comment:
Why do we need this? And not for other helpers?

Reply:
https://github.com/babel/babel/blob/c8362e3b00022c466179180e3552f2a3b71a7c80/packages/babel-helpers/src/helpers/tsconfig.json#L16

Since it was not worth migrating to strict TS, we manually excluded it.